src/Umbraco.Core/Constants-HealthChecks.cs

```diff
@@ -52,6 +52,7 @@ public static class Security
                 [Obsolete("This link is not used anymore in the XSS protected check.")]
                 public const string XssProtectionCheck = "https://umbra.co/healthchecks-xss-protection";
                 public const string ExcessiveHeadersCheck = "https://umbra.co/healthchecks-excessive-headers";
+                public const string CspHeaderCheck = "https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP";
```

Comment:
Would be nice if you (or someone else) could also add a page to the umbraco docs (like the others, e.g. https://docs.umbraco.com/umbraco-cms/extending/health-check/guides/clickjackingprotection). 🙂

Reply:
@lauraneto Totally agree, I just created an issue and I will add a page to the docs.